<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseList</class>
 <widget class="QWidget" name="ChooseList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="allElementsLabel">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>All elements:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="allElementsList">
       <property name="selectionMode">
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="addItemButton">
       <property name="toolTip">
        <string>Add</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
       <property name="arrowType">
        <enum>Qt::RightArrow</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="removeItemButton">
       <property name="toolTip">
        <string>Remove</string>
       </property>
       <property name="text">
        <string>&gt;</string>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
       <property name="arrowType">
        <enum>Qt::LeftArrow</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="selectedElementsLabel">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Selected elements:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="currentElementList">
       <property name="selectionMode">
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
